db/data/20230220095332_set_transaction_date_on_commitments.rb

```diff
@@ -0,0 +1,14 @@
+# Run me with `rails runner db/data/20230220095332_set_transaction_date_on_commitments.rb`
+
+commitments_to_update = Commitment.where(transaction_date: nil)
+
+puts "Setting transaction date on #{commitments_to_update.count} Commitments"
+
+commitments_to_update.each do |commitment|
+  print "."
```

Comment:
Should the `.` go after the `update` just in case it's unsuccessful and you want to count how many dots there are as indicators of successful transactions? 🤓

Reply:
fair point!